11-change-ppp-mtu-debug.sh

```diff
@@ -30,17 +30,18 @@ while true; do
           echo $pinterface is one we should be checking
           # Check to see if we need to update the config file
           echo Checking MTU for $pinterface
-          pmtu=$(grep $(($PTARGET-8)) /etc/ppp/peers/$pinterface)
+          pmtu=$(grep 'mtu $(($PTARGET))' /etc/ppp/peers/$pinterface)
           if [[ $pmtu ]]; then
+            echo MTU already correct in /etc/ppp/peers/$pinterface
+            pmtucorrect=1
+          else
             echo Current config file MTU for $pinterface is $pmtu
             echo Updating config file for $pinterface
             echo Making changes to /etc/ppp/peers/$pinterface
             # Update MTU in ppp interface config file
-            sed -i 's/ '$(($PTARGET-8))'/ '$PTARGET'/g' /etc/ppp/peers/$pinterface
-            killall -SIGHUP pppd
-          else
-            echo MTU already correct in /etc/ppp/peers/$pinterface
-            pmtucorrect=1
+            sed -i 's/mtu\s[0-9]*/mtu '$PTARGET'/g' /etc/ppp/peers/$pinterface
+            sed -i 's/mru\s[0-9]*/mru '$PTARGET'/g' /etc/ppp/peers/$pinterface
+            restartpppd=1
```

Comment:
i noticed when using this script that the ppp0 interface would have the wrong value after it worked, but the config would be correct. i realized it was because this was happening too soon, it needs to happen after the eth interface is updated. so i just set restartpppd=1 here and have pppd restart later instead of right away